src/tests/ExtractorTests.cs

```diff
@@ -111,9 +122,8 @@ public void TestMiniMagic(string fileName, ArchiveFileType expectedArchiveFileTy
         [DataRow("zblg.zip", true)]
         [DataRow("zbsm.zip", false)]
         [DataRow("zbsm.zip", true)]
-        //[DataRow("zbxl.zip", false)]
+        //[DataRow("zbxl.zip", false)] //These work but have stalled the anti-malware scanner on the pipeline
```

Comment:
Maybe they need our zip bomb detection code?

Reply:
Interestingly it doesn't stall running defender locally.